scripts/extract-hvd-content.ts

```diff
@@ -38,13 +36,13 @@ const BASE_REPO_CONFIG = {
 	// Skip extraction in deploy previews
 	if (isDeployPreview()) {
 		console.log(
-			'Note: Content repo deploy preview detected. Skipping HVD content.'
+			'Note: content repo deploy preview detected. Skipping HVD content.'
```

Comment:
If we're in a pr preview, we won't generate the extracted dir because we can reference data local to the repo we are in via `LOCAL_CONTENT_DIR`